lib/auth.ts

```diff
@@ -1,13 +1,15 @@
 import { expect, Page } from '@playwright/test';
-import blockAnalyticsDomains from './blocker';
 import { config } from './config';
 
 export default async function login(
   page: Page,
   username: string = config.adminUsername,
   password: string = config.adminPassword
 ) {
-  await blockAnalyticsDomains(page);
+  // Disable analytics
+  await page.addInitScript(() => {
+    window.localStorage.setItem('chrome:analytics:disable', String(true));
```

Comment:
How this is supposed to work with browsers different from chrome?

Reply:
Ah, I see the confusion. Chrome is the name of the consoledot app that loads everything. It's not the best name I agree. That string is just a namespaces key